Resources/views/PlatformUI/shell.html.twig

```diff
@@ -52,33 +52,34 @@
             var app = new Y.eZ.PlatformUIApp({
                     container: '.ez-platformui-app',
                     viewContainer: '.ez-view-container',
-                    root: "{{ app.request.attributes.get( 'semanticPathinfo' ) }}",
-                    assetRoot: "{{ asset( '/' ) }}",
-                    anonymousUserId: "{{ anonymousUserId }}",
+                    config: {{ parameters|json_encode|raw }},
```

Comment:
since the App is not yet able to understand that, I would not add this line in this PR but we'll do that in a follow up issue/PR.